### PR TITLE
OpenStackDataPlaneDeployment docs

### PR DESCRIPTION
**Pull request by slagle**
Also fixes several out of date references related to the changes for
OpenStackDataPlaneNodeSet.

### CODE REVIEW

File: docs/deploying.md [hunk 1]
```diff
@@ -79,46 +77,45 @@ This document will cover writing the `YAML` document for an
...
-example. Under the role, a `preProvisioned` field is set to `True` since these
-nodes are preprovisioned. A `nodeTemplate` field is also started that contains
-the fields that will have their values inherited by each node in the role. See
-[Inheritance](inheritance.md) for more details about how role and node
-inheritance works. Within `nodeTemplate`, the fields shown are documented
-inline in the example.
+Consider using different OpenStackDataPlaneNodeSets to logically group nodes in
+a way that makes sense. Differences between nodes are likely to include
+configurations due to hardware, location, or networking.  As differences grow,
+use different OpenStackDataPlaneNodeSets to manage similarly configure nodes
```
Comment (fultonj) [inline docs/deploying.md:95]: Please use "configured" not "configure". 

"to manage similarly configured nodes"

"more easily" can be removed

File: docs/deploying.md [hunk 2]
```diff
@@ -79,46 +77,45 @@ This document will cover writing the `YAML` document for an
...
+more easily.
+
+The `preProvisioned` field indicates that the nodes are already provisioned
+are powered on and booted into an installed operating system.
```
Comment (fultonj) [inline docs/deploying.md:99]: "are" can be removed

**Review by fultonj — COMMENTED** [review]
I'm +2 on this.

Though below I noted a few grammatical improvements.

File: docs/deploying.md (hunk 1 above)
Comment (slagle) [reply docs/deploying.md]: Done.

File: docs/deploying.md (hunk 2 above)
Comment (slagle) [reply docs/deploying.md]: Done.

**Review by fultonj — APPROVED** [review]

**Review by fao89 — COMMENTED** [review]
could you please add an entry for `openstack_dataplanedeloyment.md`
here: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/mkdocs.yml#L29-L31

**Comment by slagle** [comment]
> @slagle: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/prow/dataplane-operator-build-deploy-kuttl 	[9a7dc90](https://github.com/openstack-k8s-operators/dataplane-operator/commit/9a7dc904379dc7aa8e74a1c9a759428b41614f2f) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_dataplane-operator/410/pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl/1704537314971095040) 	true 	`/test dataplane-operator-build-deploy-kuttl`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openstack-k8s-operators&repo=dataplane-operator&pr=410). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:slagle).

`
E0920 18:07:49.538430      13 reflector.go:148] k8s.io/client-go/tools/watch/informerwatcher.go:146: Failed to watch *v1.Build: failed to list *v1.Build: Get "https://172.30.0.1:443/apis/build.openshift.io/v1/namespaces/ci-op-vhjc7cl6/builds?fieldSelector=metadata.name%3Dopenstack-operator-ci-build-deploy-kuttl&resourceVersion=61037642": dial tcp 172.30.0.1:443: connect: no route to host
`

**Comment by slagle** [comment]
> could you please add an entry for `openstack_dataplanedeloyment.md` here: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/mkdocs.yml#L29-L31

Done.

**Review by fao89 — APPROVED** [review]

**Pull request merged**